WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsActivity.java

```diff
@@ -237,42 +259,24 @@ protected Dialog onCreateDialog(int id) {
     }
 
     @Override
-    public void onModerateComment(final SiteModel site, final Comment comment,
+    public void onModerateComment(final SiteModel site, final CommentModel comment,
                                   final CommentStatus newStatus) {
         FragmentManager fm = getFragmentManager();
         if (fm.getBackStackEntryCount() > 0) {
             fm.popBackStack();
         }
 
         if (newStatus == CommentStatus.APPROVED || newStatus == CommentStatus.UNAPPROVED) {
-            getListFragment().setCommentIsModerating(comment.commentID, true);
+            // getListFragment().setCommentIsModerating(comment.getRemoteCommentId(), true);
```

Comment:
Should these commented-out snippets be removed (there are a few more cases further down)? Otherwise we should add a TODO here explaining.